api/constants/constants.go

```diff
@@ -124,7 +124,22 @@ const (
 	// RemoteAuthServer is a special non-resolvable address that indicates client
 	// requests a connection to the remote auth server.
 	RemoteAuthServer = "@remote-auth-server"
+)
 
-	// FileExtSSHCert is a file extension used for Certificate files
+const (
+	// SessionKeyDir is the sub-directory where session keys are stored (.tsh/keys).
+	SessionKeyDir = "keys"
+	// FileNameKnownHosts is a file that stores known hosts.
+	FileNameKnownHosts = "known_hosts"
+	// FileExtTLSCert is the filename extension/suffix of TLS certs
+	// stored in a profile (./tsh/keys/profilename/username-x509.pem).
+	FileExtTLSCert = "-x509.pem"
+	// FileNameTLSCerts is the filename of Cert Authorities stored in a
+	// profile (./tsh/keys/profilename/certs.pem).
+	FileNameTLSCerts = "certs.pem"
+	// FileExtCert is a file extension used for SSH Certificate files.
 	FileExtSSHCert = "-cert.pub"
+	// FileExtPub is a file extension used for SSH Certificate Authorities
+	// stored in a profile (./tsh/keys/profilename/username.pub).
+	FileExtPub = ".pub"
 )
```

Comment:
These constants aren't really used in the `api` module except for tests. Is the idea to gradually move everything from under `lib/client` to `api/client`?

Reply:
These are also used in `profile.go` in `profile.TLSConfig()` and `profile.SSHClientConfig()`. We only want to move things that are essential to the API, but the majority of `lib/client` seems to be internal logic. I don't expect much more to be moved into `api/client`.